react/src/lib/components/SubsurfaceViewer/layers/axes2d/axes2DLayer.ts

```diff
@@ -171,7 +171,8 @@ export default class Axes2DLayer extends Layer<Axes2DLayerProps<unknown>> {
         min: number,
         max: number,
         viewSide: ViewSide,
-        pixel2world: number
+        pixel2world: number,
+        zoom_scale: number
```

Comment:
Please check variable case consistency. I believe camel case is the most common style in this project. Perhaps there is an eslint rule that we should enable?